index.html

```diff
@@ -59,36 +63,71 @@
         background-size: cover;
         background-position: center;
         color: #fff;
+        margin-bottom: 75px;
       }
 
       .hero .container {
         flex: 0 1 1000px;
         display: flex;
-        flex-flow: column nowrap;
-        align-items: start;
+        flex-flow: row nowrap;
+        align-items: flex-end;
+        margin: 0;
       }
 
-      .hero p {
+      .hero .hero-quote {
+        margin-bottom: -100px;
+      }
+
+      .hero p.quote {
+        font-size: 1.5rem;
+        line-height: 1.25;
+        padding: 8px 0;
+        border-left: 20px solid #ffcb25;
+        max-width: 14em;
         margin: 0;
-        background: white;
-        color: black;
-        padding: 8px 14px;
-        font-size: 1.4em;
       }
 
-      .hero .quote {
-        display: flex;
-        flex-flow: column nowrap;
-        align-items: flex-start;
+      .hero p.quote .highlight {
+        background-color: #ffcb25;
+        padding: 9px 0;
+        color: #202020;
+        display: inline;
+        margin: 0;
+        font-weight: 600;
       }
 
-      .hero .quote p:not(:first-child) {
-        padding-top: 0;
-        margin-top: -8px;
+      .hero p.quote .highlight .offset {
+        position: relative;
+        left: -10px;
       }
 
       .hero p.quote-from {
-        font-size: 0.9em;
+        font-size: 0.875rem;
+        line-height: 1;
+        padding: 8px 10px;
+        margin: 0;
+        display: inline-block;
+        background: #202020;
+        color: #ffcb25;
+        font-weight: 300;
```

Comment:
A lot of fonts don't have 300, and makes it really hard to read.

Reply:
This is Cantarell 
![image](https://user-images.githubusercontent.com/9994172/46985318-8c2c8580-d135-11e8-9677-eb8dc241d895.png)